components/Logo/iustlogo.tsx

```diff
@@ -0,0 +1,12 @@
+import Image from 'next/image';
+import Link from 'next/link';
+const iustlogo = () => {
+  return (
+    <div className='logo'>
+      <a href='google.com'>
+        <Image src='/iustlogo.png' alt='missing' width={200} height={65} />
+      </a>
+    </div>
+  );
+};
+export default iustlogo;
```

Comment:
This should not be called iustlogo, Component names are always generic, you can call this Logo.
All components should use PascalCase